protobuf/module_msg.proto

```diff
@@ -972,3 +972,17 @@ message VXLANEncapArg {
 message WildcardMatchArg {
   repeated WildcardMatchField fields = 1; /// A list of WildcardMatch fields.
 }
+
+/**
+ * The ARP Responder module is responding to ARP requests and can optionally learn new MAC's-IP's mapping
```

Comment:
The comment "can optionally learn ..." can be updated so as to be clear that it is not supported yet...? For example, "TODO:"

Q: is the ARP learning feature for proxy ARP?